tutorials/Data_sync.ipynb

```diff
@@ -81,11 +82,10 @@
     "### Setting sync options\n",
     "`sync` takes various options:\n",
     "\n",
-    "- `ignore_checksum`: sync compares the checksum values and file sizes of the source and target files to determine whether synchronization is needed. If the `ignore_checksum` option is set to True, only the file size (instead of the the size and checksum value) is used for determining whether synchronization is needed. This mode gives a potentially faster operation but the result is less accurate.\n",
     "- The `max_level` option controls the depth up to which the file tree will be synchronized. With `max_level` set to None (default), there is no limit (full recursive synchronization). A max level of 1 synchronizes only the source's root, max level 2 also includes the first set of subfolders/subcollections and their contents, etc.\n",
     "- The `copy_empty_folders` (default False) option controls whether folders/collections that contain no files or subfolders/subcollections will be synchronized.\n",
     "- The `dry_run` option lists all the source files and folders that need to be synchronized without actually performing the synchronization.\n",
-    "- The `verify_checksum` (default True) option will calculate and verify the checksum on the data after up- or downloading. A checksum mismatch will generate an error, but will not abort the synchronization process."
+    "- The `verify_checksum` (default True) option will calculate and verify the checksum on the data after up- or downloading. A checksum mismatch will generate an error, aborting the synchronization process."
```

Comment:
Maybe it would be nice to give some instruction on how to deal with this situation?